Makefile

```diff
@@ -7,9 +7,7 @@ LAST_TAG := "$(shell git rev-list --tags --max-count=1)"
 OPMS_VERSION := "$(shell git describe --tags ${LAST_TAG})-next"
 PROTON_COMMIT := "6fa8d49d26bfd7aabc0978de6e9039bb35732695"
 
-all: build
```

Comment:
this was added so that the default command starts as `build` when you just run `make`. We can either use 
```
.DEFAULT_GOAL := build
```
but simply removing it doesn't look good to me.